challenges/backend/backend.py

```diff
@@ -0,0 +1,30 @@
+from flask import Flask, jsonify, request
+import json
+app = Flask(__name__)
+
+data_list = []
+
+# TODO: better (more verbose) errors
+@app.route('/data', methods=['GET', 'POST', 'PATCH'])
```

Comment:
How did you decide on what methods are appropriate here? For example would it make sense to include PUT or DELETE?

Reply:
I decided to implement only the requested methods, but with the current model I don't think PUT makes sense. If we had given the ability to create multiple data records, then PUT would be used to create new ones or overwrite existing ones. DELETE could be used in the current model to remove the only list.